DependencyInjection/Compiler/CommandHandlerPass.php

```diff
@@ -19,25 +23,113 @@ class CommandHandlerPass implements CompilerPassInterface
      */
     public function process(ContainerBuilder $container)
     {
-        if (!$container->has('tactician.handler.locator.symfony')) {
-            throw new \Exception('Missing tactician.handler.locator.symfony definition');
-        }
-
-        $handlerLocator = $container->findDefinition('tactician.handler.locator.symfony');
-
-        $mapping = [];
+        $defaultBusId = $this->getDefaultBusId($container);
+        $busIds = $this->getBusIds($container);
+        $busIdToHandlerMapping = [];
 
         foreach ($container->findTaggedServiceIds('tactician.handler') as $id => $tags) {
+
             foreach ($tags as $attributes) {
                 if (!isset($attributes['command'])) {
                     throw new \Exception('The tactician.handler tag must always have a command attribute');
                 }
 
-                $mapping[$attributes['command']] = $id;
+                if (array_key_exists('bus', $attributes)) {
+                    $this->abortIfInvalidBusId($attributes['bus'], $busIds);
```

Comment:
Reverted this to only check if bus is explicitly specified since otherwise it comes from the list of buses in the config which can't be wrong